### PR TITLE
Add link to consistency guidelines in README

### PR DESCRIPTION
**Pull request by IAmHughes**
This is a quality of life add to the README to link directly to the consistency fundamentals section of the guidelines.

My thought was to specifically call out the consistency portion as I feel it helps in navigating through the longer documentation.

I recognized that the contributing guidelines state to open the PR against `master`, though the default branch is `vNext`. Please let me know if I should open a PR against `vNext` instead and I'll update the contributing guidelines while doing so.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Microsoft REST API Guidelines
-The [Microsoft REST API Guidelines](Guidelines.md), as a design principle, encourages application developers to have resources accessible to them via a RESTful HTTP interface. To provide the smoothest possible experience for developers on platforms following the Microsoft REST API Guidelines, REST APIs SHOULD follow consistent design guidelines to make using them easy and intuitive.
+The [Microsoft REST API Guidelines](Guidelines.md), as a design principle, encourages application developers to have resources accessible to them via a RESTful HTTP interface. To provide the smoothest possible experience for developers on platforms following the Microsoft REST API Guidelines, REST APIs SHOULD follow [consistent design guidelines](https://github.com/IAmHughes/api-guidelines/blob/vNext/Guidelines.md#7-consistency-fundamentals) to make using them easy and intuitive.
```
Comment (gareth-a-jones) [inline README.md:2]: Relative link? Or am I misunderstanding how GitHub handles links?

**Comment by gareth-a-jones** [comment]
Yes, please do port it to vNext.  We're only taking contributions against that for now.  I just updated the contribution guideline.  Thanks for the spot.

File: README.md (hunk 1 above)
Comment (IAmHughes) [reply README.md:2]: yes this is a relative link to the header within the `Guidelines.md` doc.

**Comment by IAmHughes** [comment]
Changed base from `master` to `vNext` per @garethj-msft's [feedback](https://github.com/Microsoft/api-guidelines/pull/134#issuecomment-428681980)

File: README.md (hunk 1 above)
Comment (gareth-a-jones) [reply README.md:2]: So weird - I get to see it as an absolute URL to your fork of the repo.

Comment (gareth-a-jones) [reply README.md:2]: I merged it and it still was absolute so I hand fixed it after the fact to be relative.

Comment (IAmHughes) [reply README.md:2]: Weird! Sorry about that @garethj-MSFT, thank you!

**Pull request merged**